gulpfile.js

```diff
@@ -117,10 +122,17 @@ gulp.task('dev', function (cb) {
 
     runSequence('build-dev', 'server', cb);
 });
+
 gulp.task('prod', function (cb) {
     WATCH = true;
 
     runSequence('build-prod', 'server', cb);
 });
 
+gulp.task('ci', ['build-dev'], function () {
+    return gulp.src(['dist/**', '!dist/client.zip'])
+        .pipe(zip('client.zip'))
+        .pipe(gulp.dest('dist'));
+});
+
```

Comment:
We should be starting with a clean slate on each build, but it probably wouldn't hurt to have a clean task anyway.  I can add that as a separate change.